docs/maps/search.asciidoc

```diff
@@ -2,35 +2,69 @@
 [[maps-search]]
 == Search geographic data
 
-Search across the layers in your map to focus in on just the data you want.
+Search across the layers in your map to focus on just the data you want. Combine free text search with field-based search using the <<kuery-query,{kib} Query Language>>. Set the time filter to restrict layers by time.
 
-Layers that request data from {es} are narrowed when you submit a <<kibana-concepts-searching-your-data, search request>>.
+This image shows an example of global search and global time filter narrowing results.
+
+[role="screenshot"]
+image::maps/images/global_search_bar.png[]
+
+Only layers requesting data from {es} are narrowed by global search and global time filter.
+To add a layer that requests data from {es} to your map, click *Add layer*, then select one of the following:
+
+* Documents
+
+* <<terms-join, Choropleth>>
+
+* Clusters and grid
+
+* <<heatmap-layer, Heat map>>
+
+* Point to point
+
+* Top hits per entity
+
+* Tracks
+
+[float]
+[[maps-narrow-layer-by-global-search]]
+==== Narrow layers by global search
+
+Layers that request data from {es} are narrowed when you submit a <<kibana-concepts-searching-your-data, search>>.
 Layers narrowed by semi-structured search and filters contain the filter icon image:maps/images/filter_icon.png[] next to the layer name in the legend.
 
-Layers that request data from {es} using an <<index-patterns, index pattern>> with a configured time field are narrowed by the <<set-time-filter, global time filter>>.
-These layers contain the clock icon image:maps/images/clock_icon.png[clock icon] next to the layer name in the legend. Use the time slider to quickly select time slices within the global time filter range.
-Click previous and next buttons to advance the time slice backward or forward. Click play to animate your spatial temporal data.
+You can prevent the global search from applying to a layer by configuring the following:
 
-You can create a layer that requests data from {es} from the following:
+* In *Filtering*, clear the *Apply global filter to layer data* checkbox to turn off global search for the layer source.
```

Comment:
If both of these need to be done, use numbered steps. Otherwise it sounds like you must do one or another.

Reply:
Both don't always need to be done. You may do one or the other depending on how you want to limit impact of global search on the layer. I will leave as is for now and we can revisit if its confusing.